### ISSUE TITLE
The PropGet trait has not been re-exported

### ISSUE THREAD

**Issue by robbert-vdh** [vizia/vizia#86]
Like `PropSet` makes setting styling properties much simpler, `PropGet` makes it a lot simpler to write widgets that need to be aware of their styling (for instance, to be able to subtract their borders from the total size, or to respect some common styling properties while doing custom rendering on a canvas). It would be nice to be able to use this for custom widgets.

**Comment by geom3trik** [comment]
I beleieve this issue may be fixed by the addition of the `EventContext` and `DrawContext` in #178 and #189. The style properties can be set on the current entity with the relevant methods on the `EventContext` and retrieved for custom drawing with the relevant methods on `DrawContext`.

**Comment by robbert-vdh** [comment]
Yes the new context methods indeed work great for the `DrawContext`. In the `EventContext` you can get the information as of #237, but there are no direct getters like `PropGet` provided.

**Comment by geom3trik** [comment]
> Yes the new context methods indeed work great for the `DrawContext`. In the `EventContext` you can get the information as of #237, but there are no direct getters like `PropGet` provided.

You make a good point. I think it would be useful to be able to both get and set properties from the `EventContext`. Not sure if I'll go with a trait or not but my thinking is something like `property_name` for gettng and `set_property_name` for setting. So for example:
```rust
let background_color = cx.background_color().copied().unwrap_or_default();
cx.set_background_color(Color::red());
```
